blocks/bricks/recurrent.py

```diff
@@ -59,27 +59,24 @@ def initial_state(self, state_name, batch_size, *args, **kwargs):
 def recurrent(*args, **kwargs):
     """Wraps an apply method to allow its iterative application.
 
-    This decorator allows you to use implementation of an RNN
-    transition to process sequences without writing the
-    iteration-related code again and again. In the most general form
-    information flow of a recurrent network can be described as
-    follows: depending on the context variables and driven by input
-    sequences the RNN updates its states and produces output sequences.
-    Thus the input variables of your transition function play one of
-    three roles: an input, a context or a state. These roles should be
-    specified in the method's signature to make iteration possible.
+    This decorator allows you to use implementation of an RNN transition to
```

Comment:
I'm not following this sentence. Maybe "This decorator allows you to implement a recurrent transition to process sequences..."?